extensions/amp-story/1.0/amp-story-system-layer.css

```diff
@@ -53,6 +53,78 @@
   opacity: 0 !important;
 }
 
+.i-amphtml-story-story-attribution {
+  top: 0 !important;
+  left: 0 !important;
+  position: absolute;
+  border: none !important;
+  pointer-events: auto !important;
+  cursor: pointer !important;
+  max-width: 50% !important;
+  padding-top: 6px !important;
+  height: 48px !important;
+  place-items: center !important;
+  display: flex !important;
+  text-decoration: none !important;
+  visibility: hidden !important;
+}
+
+[dir=rtl] .i-amphtml-story-story-attribution {
+  left: auto !important;
+  right: 0 !important;
+}
+
+.i-amphtml-story-story-attribution > * {
+  padding-left: 6px !important;
+}
+
+[dir=rtl] .i-amphtml-story-story-attribution > * {
+  padding-left: 0 !important;
+  padding-right: 6px !important;
+}
+
+.i-amphtml-story-desktop-panels .i-amphtml-story-story-attribution > * {
+  padding-left: 8px !important;
+}
+
+[dir=rtl].i-amphtml-story-desktop-panels .i-amphtml-story-story-attribution > * {
+  padding-left: 0 !important;
+  padding-right: 8px !important;
+}
+
+.i-amphtml-story-logo-container {
+  display: grid !important;
+  place-items: center !important;
+}
+
+[desktop] .i-amphtml-story-logo-container {
+  margin-left: 4px !important;
+}
+
+[dir=rtl][desktop] .i-amphtml-story-logo-container {
+  margin-left: 0 !important;
+  margin-right: 4px !important;
+}
+
+.i-amphtml-story-logo-container img {
+  border-radius: 100% !important;
+  border: 1px solid #D4D4D4 !important;
+  width: 32px !important;
+  height: 32px !important;
+  object-fit: cover !important;
+  box-shadow: 0px 2px 6px rgba(0, 0, 0, 0.16) !important;
```

Comment:
Are you re-using these numbers or coming up with new ones? Can we re-use something we already have?

Reply:
Reused same numbers we used for other shadows inside the system layer.